postcss.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  plugins: [require('autoprefixer'), require('cssnano')],
```

Comment:
Will using cssnano for every build have any impact on build time or the css that gets produced? Not familiar with it, but seems like it had only been used on optimized builds before

Reply:
After speaking with @timwright12, it will only impact the build time on the not optimized builds

Reply:
As long as the time penalty isn't too much, and there aren't any sourcemap issues based on breaking changes, that seems fine to me. I'll defer to Tim on the review though since he might have additional knowledge.